src/magic_folder/list.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Least Authority TFA GmbH
+# See COPYING for details.
+
+"""
+Implements ```magic-folder list``` command.
+"""
+
+import json
+
+from twisted.internet.defer import (
+    inlineCallbacks,
+)
+
+from .client import (
+    create_magic_folder_client,
+)
+
+
+@inlineCallbacks
+def magic_folder_list(config, output, as_json=False, include_secret_information=False):
+    """
+    List folders associated with a node.
+
+    :param GlobalConfigDatabase config: our configuration
+
+    :param output: a file-like object to which the output will be written
+
+    :param bool as_json: return answer in JSON
+
+    :param bool include_secret_information: include sensitive private
+        information (such as long-term keys) if True (default: False).
+
+    :return: JSON response from `GET /v1/magic-folder`.
+    """
+    from twisted.internet import reactor
```

Comment:
I'd suggest reactor as a parameter.  Let the CLI entrypoint import the global reactor and require everything else to receive it as a parameter.